content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -12,12 +12,26 @@ export class _StartupOverlay extends React.PureComponent {
     this.initScene = this.initScene.bind(this);
     this.removeOverlay = this.removeOverlay.bind(this);
     this.onInputInvalid = this.onInputInvalid.bind(this);
+    this.setFormFields = this.setFormFields.bind(this);
 
-    this.state = {emailInput: ""};
+    this.state = {emailInput: "", flowId: "", flowBeginTime: ""};
     this.initScene();
   }
 
   initScene() {
+    let req = new XMLHttpRequest();
```

Comment:
Is there any reason you can't use the regular fetch API here instead of XMLHttpRequest? https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API